kahuna/public/js/image/view.html

```diff
@@ -223,7 +223,7 @@
             </button>
 
             <div ng:show="metadataExpanded" class="metadata metadata-line image-info__group--dl">
-                <dl ng:repeat="(key, value) in ctrl.metadata" ng:if="ctrl.isUsefulMetadata(key)" class="metadata__body image-info__group--dl">
+                <dl ng:repeat="(key, value) in ctrl.spacedMetadata" ng:if="ctrl.isUsefulMetadata(key)" class="metadata__body image-info__group--dl">
                     <dt class="metadata-line__key image-info__group--dl__key--full-metadata">{{key}}</dt>
```

Comment:
Alternatively since it's really just a view concern, the formatting could be done as a filter on here (`{{key | spacedWords}}`), there are other examples of filters if you want to try it this way.